index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
 
-<p><dfn data-lt="locale fallback|fallback|Locale fallback" id="def_locale_fallback" class="lint-ignore export">Locale fallback</dfn>. The process of searching for translated content, locale data, or other resources by "falling back" from more-specific resources to more-general ones following a deterministic pattern.</p>
+<p><dfn class="lint-ignore export">Locale fallback</dfn>. The process of searching for translated content, locale data, or other resources by "falling back" from more-specific resources to more-general ones following a deterministic pattern.</p>
 
-<p><dfn data-lt="locale-neutral|locale neutral|Locale-neutral" id="def_locale_neutral" class="lint-ignore export">Locale-neutral</dfn>. A <a href="#def_non_linguistic_field" class="termref">non-linguistic field</a> is said to be <a href="#def_locale_neutral" class="termref">locale-neutral</a> when it is stored or exchanged in a format that is not specifically appropriate for any given language, locale, or culture and which can be interpreted unambiguously for presentation in a <a href="#def_locale_aware" class="termref">locale aware</a> way.</p>
+<p><dfn data-lt="locale neutral" class="lint-ignore export">Locale-neutral</dfn>. A [=non-linguistic field=] is said to be locale-neutral when it is stored or exchanged in a format that is not specifically appropriate for any given language, locale, or culture and which can be interpreted unambiguously for presentation in a [=locale-aware=] way.</p>
 
-<p><dfn id="def_localizable_text" class="lint-ignore export">Localizable text</dfn>.  Document contents and strings intended as human-readable text and <b>not</b> to any of the surrounding or embedded syntactic content that form part of the document structure. Note that syntactic content can have localizable text embedded in it, such as when an [[HTML]] <code class="kw">img</code> element has an <code class="kw">alt</code> attribute containing a description of the image.  <!-- REINSTATE THIS AFTER CHARMOD-NORM HAS BEEN UPDATED  [[CHARMOD-NORM]] gives <a href="https://www.w3.org/TR/charmod-norm/#dfn-localizable-content">some additional examples</a>.-->  See also <a class="termref" href="#def_syntactic_content">syntactic content</a> and <a class="termref" href="#def_natural_language">natural language</a>.</p>
+<p><dfn class="lint-ignore export">Localizable text</dfn>.  Document contents and strings intended as human-readable text and <b>not</b> to any of the surrounding or embedded syntactic content that form part of the document structure. Note that syntactic content can have localizable text embedded in it, such as when an [[HTML]] <code class="kw">img</code> element has an <code class="kw">alt</code> attribute containing a description of the image.  <!-- REINSTATE THIS AFTER CHARMOD-NORM HAS BEEN UPDATED  [[CHARMOD-NORM]] gives <a href="https://www.w3.org/TR/charmod-norm/#dfn-localizable-content">some additional examples</a>.-->  See also [=syntactic content=] and [=natural language=].</p>
```

Comment:
I think we can re-instate the commented out text, since it is the TR version of charmod?
```suggestion
<p><dfn class="lint-ignore export">Localizable text</dfn>.  Document contents and strings intended as human-readable text and <b>not</b> to any of the surrounding or embedded syntactic content that form part of the document structure. Note that syntactic content can have localizable text embedded in it, such as when an [[HTML]] <code class="kw">img</code> element has an <code class="kw">alt</code> attribute containing a description of the image.  [[CHARMOD-NORM]] gives <a href="https://www.w3.org/TR/charmod-norm/#dfn-localizable-content">some additional examples</a>. See also [=syntactic content=] and [=natural language=].</p>
```

Reply:
Done.